sit-awareness/test.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import argparse
+import cv2
+from cv import PoseDetector
+
+# "python test.py 0" to run from camera in port 0
+# "python test.py video.mp4" to run from the video recording video.mp4
+
+parser = argparse.ArgumentParser()
+parser.add_argument("input", help="read from the given camera or file", default="0")
+parser.add_argument("target", help="target to detect pose for", default="lb")
```

Comment:
Again, we should have enough memory to spell out lb and pp. Also, in cases where there is a fixed number of accepted inputs you can use the [choices option](https://docs.python.org/3/library/argparse.html#choices) to check the input.